dev/NavigationView/NavigationView.cpp

```diff
@@ -4521,7 +4521,14 @@ void NavigationView::UpdatePaneShadow()
         shadowReceiver.HorizontalAlignment(winrt::HorizontalAlignment::Left);
 
         // Ensure shadow is as wide as the pane when it is open
-        shadowReceiver.Width(OpenPaneLength());
+        if (DisplayMode() == winrt::NavigationViewDisplayMode::Compact)
+        {
+            shadowReceiver.Width(OpenPaneLength());
+        }
+        else
+        {
+            shadowReceiver.Width(OpenPaneLength() - shadowReceiverMargin.Right);
```

Comment:
>- shadowReceiverMargin.Right [](start = 50, length = 28)

Why isn't this needed in the Compact case?

Reply:
For some reason in the case of Compact, the sizing seems to be calculated correctly. In the case of Minimal it is not ...

Reply:
hmm any theories why?

Reply:
My guess is that the negative margin is the issue for minimal. This blue area is the shadow receiver (reports itsself as having an actualwidth of 320 which is the same as the pane. The yellow area is the margin (0,**-32**,**-32**,**-32**):

![image](https://user-images.githubusercontent.com/16122379/90449014-d2165400-e0e6-11ea-81f3-3659efd375ac.png)

This is how it looks without that margin:

![image](https://user-images.githubusercontent.com/16122379/90449060-e9edd800-e0e6-11ea-9196-6972de3756c5.png)

In both cases it reports the same actual size. So I think the negative margin is just not reacting correctly here.

Reply:
What is the margin from the compact case?

Reply:
Exactly the same.

Reply:
I still wish I understood this better. I'm hoping there is some comment we can include in the code explaining why the distinction is needed.

Reply:
The only thing I can say with confidence is that in theory, the sizing is actually correct:
The shadow pane receiver has 320px width (actual width!), the same is true for the pane. In theory, having a negative right margin for the shadow receiver would allow the shadow to be visible as the shadow receiver gets extended to the right, but that is not the case for some reason.